ports/esp32/machine_pin.c

```diff
@@ -290,7 +290,11 @@ STATIC mp_obj_t machine_pin_obj_init_helper(const machine_pin_obj_t *self, size_
 
     #if CONFIG_IDF_TARGET_ESP32C3
     if (self->id == 18 || self->id == 19) {
+        #if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(4, 4, 1)
         CLEAR_PERI_REG_MASK(USB_DEVICE_CONF0_REG, USB_DEVICE_USB_PAD_ENABLE);
+        #else
+        CLEAR_PERI_REG_MASK(USB_SERIAL_JTAG_CONF0_REG, USB_SERIAL_JTAG_USB_PAD_ENABLE);
+        #endif
```

Comment:
Can we just use the new constants unconditionally, ie for all IDF versions (that support C3)?  These JTAP consts do exist in IDF 4.3.2 at least.

Reply:
That is better indeed. Did not want to break anything for older IDF versions, but that is indeed not an issue as the minimum version that works with the C3 is recent enough already to have these constants.

Tested and pushed.